### PR TITLE
raidboss: normalize timelines in preparation for conversion

### PR DESCRIPTION
**Pull request by quisquous**
- Various sync fixes where it is clearly broken.
- Various sync massaging to look like other syncs.
- Break apart any (source1:id1|source2:id2) into (source1|source2):(id1:id2)
- Adjust LakshmiEx timeline to avoid sync /.*/ by adding filler.

### CODE REVIEW

**Comment by quisquous** [comment]
These are all manual changes, so need a careful eye. @JLGarber do you have any time to do a review of this and double check my work?

This is all to make #5977 work more easily.

File: ui/raidboss/data/03-hw/raid/a3n.txt [hunk 1]
```diff
@@ -9,59 +9,59 @@ hideall "--sync--"
...
+11.6 "Splash x3" duration 3 #sync / 1[56]:[^:]*:Living Liquid:12EF:/
+19.9 "Protean Wave" sync / 1[56]:[^:]*:Living Liquid:12F0:/
+23.5 "Fluid Swing" sync / 1[56]:[^:]*:Living Liquid:12EE:/
+27.6 "Splash x3" duration 3 #sync / 1[56]:[^:]*:Living Liquid:12EF:/
+32.9 "Fluid Swing" sync / 1[56]:[^:]*:Living Liquid:12EE:/
+39.0 "Sluice" sync / 1[56]:[^:]*:Living Liquid:12F2:/
+42.0 "--sync--" sync / 1[56]:[^:]*:Living Liquid:12F3:/
+44.0 "--sync--" sync / 1[56]:[^:]*:Living Liquid:12F4:/
+47.2 "Fluid Swing" sync / 1[56]:[^:]*:Living Liquid:12EE:/
+53.3 "Splash x3" duration 3 #sync / 1[56]:[^:]*:Living Liquid:12EF:/
```
Comment (JLGarber) [inline ui/raidboss/data/03-hw/raid/a3n.txt:22]: I'm genuinely surprised these weren't correct at the time. I updated `make_timeline` something like two weeks before this was merged!

File: ui/raidboss/data/04-sb/raid/o6s.txt [hunk 2]
```diff
@@ -93,11 +93,11 @@ hideall "--sync--"
...
 603 "Possession" sync / 1[56]:[^:]*:Demon Chadarnook:2803:/ window 20
-611 "Easterlies" sync / 03:........:Easterly:/ 
+611 "Easterlies" sync / 03:........:Easterly:/
 613 "Rock Hard" sync / 1[56]:[^:]*:Portrayal of Earth:2812:/
 614 "Demonic Wave" sync / 1[56]:[^:]*:Portrayal of Water:2830:/
 622 "Demonic Howl" sync / 1[56]:[^:]*:Demon Chadarnook:282B:/
 626 "Featherlance" sync / 1[56]:[^:]*:Easterly:283E:/
 
-626 "--sync--" sync / 1[56]:[^:]*:284D:Demon Chadarnook:/
+626 "--sync--" sync / 1[56]:[^:]*:Demon Chadarnook:284D:/
```
Comment (JLGarber) [inline ui/raidboss/data/04-sb/raid/o6s.txt:102]: This looks suspiciously like a `StartsUsing` sync that was mislabeled. I'm fairly sure I screwed this up during the Endwalker conversion. https://github.com/quisquous/cactbot/blame/beb375702898ee4da6d4e7ef4320d080949fef81/ui/raidboss/data/04-sb/raid/o6s.txt#L101 from right before that conversion has that with the very old `StartsUsing` syntax.

File: ui/raidboss/data/05-shb/trial/hades.txt [hunk 3]
```diff
@@ -92,7 +92,7 @@ hideall "Double"
 903.0 "--sync--" sync / 1[56]:[^:]*:Hades:417F:/
 905.0 "Gaol Add"
 906.5 "--sync--" sync / 14:[^:]*:Hades:416C:/
-936.2 "Chorus Of The Lost" sync / 1[56]:[^:]*:416C:Hades:/
+936.2 "Chorus Of The Lost" sync / 1[56]:[^:]*:Hades:416C:/
```
Comment (JLGarber) [inline ui/raidboss/data/05-shb/trial/hades.txt:95]: Same situation here, I think this should be 14: etc.

File: ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.txt [hunk 4]
```diff
@@ -191,7 +191,7 @@ hideall "--sync--"
 982.6 "Fate Calibration α" sync / 1[56]:[^:]*:Perfect Alexander:487C:/
 985.6 "--untargetable--"
 989.8 "Ordained Motion/Stillness" sync / 1[56]:[^:]*:Perfect Alexander:49A[BC]:/
-992.8 "Obloquy, Solidarity and 3x Severity" sync  /:Perfect Alexander:4861:/
+992.8 "Obloquy, Solidarity and 3x Severity" sync / 1[56]:[^:]*:Alexander Prime:4861:/
```
Comment (JLGarber) [inline ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.txt:194]: I didn't know Alexander Prime showed up to do random stuff in phase 4! That's wild.

**Review by JLGarber — APPROVED** [review]

File: ui/raidboss/data/05-shb/trial/hades.txt (hunk 3 above)
Comment (quisquous) [reply ui/raidboss/data/05-shb/trial/hades.txt:95]: I think this one seems unlikely, as there is a 14 line at 906.5, and 906.5/936.2 parallels the later 1068.5/1098.2.

File: ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.txt (hunk 4 above)
Comment (quisquous) [reply ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.txt]: Fixed!

File: ui/raidboss/data/04-sb/raid/o6s.txt (hunk 2 above)
Comment (quisquous) [reply ui/raidboss/data/04-sb/raid/o6s.txt]: Oooooh good catch, fixed.

File: ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.txt (hunk 4 above)
Comment (JLGarber) [reply ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.txt]: Oh. I'm slightly disappointed that wasn't a genuine random appearance.

**Pull request merged**